src/client/pages/Payment/Payment.tsx

```diff
@@ -71,7 +71,7 @@ export const PaymentPage = () => {
           </HeaderBack>
         </PageHeader>
 
-        <Content y={{ base: 2, largeScreen: 3 }}>
+        <Content y={{ base: 2.5 }}>
```

Comment:
nit: this can be simplified as `y={2.5}`

Reply:
tack tack